package.json

```diff
@@ -101,6 +101,12 @@
       "default": false,
       "order": 5
     },
+    "enableSynctex": {
+      "title": "Enable SyncTeX",
```

Comment:
Should it? I added a title to display proper casing. Like this:
![2016-06-05-132017_136x33_scrot](https://cloud.githubusercontent.com/assets/5445049/15807057/b99259ea-2b20-11e6-8611-75133574ae8d.png)
Changing it to description yields this:
![2016-06-05-131858_126x47_scrot](https://cloud.githubusercontent.com/assets/5445049/15807058/bd9600b4-2b20-11e6-8666-8989981d7a25.png)